lib/infrastructure/player_repository.dart

```diff
@@ -12,14 +12,14 @@ class _PlayerRepository extends PlayerController {
 
   @override
   Future play(String fileName) async {
-    if (!_isSilent) {
+    if (!_isSound) {
```

Comment:
Se want to play sound if the var is true.
Remove the !

Reply:
Updated. 
I think now everything works perfectly :)